src/umd/ReactDOMUMDEntry.js

```diff
@@ -11,24 +11,24 @@
 
 'use strict';
 
+var React = require('React');
```

Comment:
I don't think this is safe... doesn't it result in React getting packaged into the ReactDOM bundle? I would expect `build/react-dom.js` just exploded in size.